x-pack/plugins/beats_management/public/pages/main/index.tsx

```diff
@@ -88,27 +96,41 @@ class MainPagesComponent extends React.PureComponent<MainPagesProps, MainPagesSt
       // },
       {
         id: '/overview/tags',
-        name: 'Configuration tags',
+        name: (
+          <FormattedMessage
+            id="xpack.beatsManagement.beats.configurationTagsTabTitle"
+            defaultMessage="Configuration tags"
+          />
+        ),
         disabled: false,
       },
     ];
 
     const walkthroughSteps = [
       {
         id: '/overview/initial/beats',
-        name: 'Enroll Beat',
+        name: intl.formatMessage({
+          id: 'xpack.beatsManagement.enrollBeat.enrollBeatStepLabel',
+          defaultMessage: 'Enroll Beat',
+        }),
         disabled: false,
         page: EnrollBeatPage,
       },
       {
         id: '/overview/initial/tag',
-        name: 'Create tag',
+        name: intl.formatMessage({
+          id: 'xpack.beatsManagement.enrollBeat.createTagStepLabel',
+          defaultMessage: 'Create tag',
+        }),
         disabled: false,
         page: CreateTagPageFragment,
       },
       {
         id: '/overview/initial/finish',
-        name: 'finish',
+        name: intl.formatMessage({
+          id: 'xpack.beatsManagement.enrollBeat.finishStepLabel',
+          defaultMessage: 'Finish',
```

Comment:
@mattapperson do you mind that I capitalized the first letter to be consistent with `Enroll Beat` and `Create tag` steps?

Reply:
No, not at all!